lib/reporter/util.js

```diff
@@ -28,7 +28,22 @@ function hasFailures(modules) {
 
 function sanitizeOutput(output, delimiter, xml) {
   if (!output || output === '') return '';
-  return _(output.split(/[\r\n]+/)).filter(function (item) {
+  /* eslint-disable no-control-regex */
+  // ([valid-range])|. matches anything but only captures valid-range
+  var validUtf8 = new RegExp('(' + [
```

Comment:
I'm not sure this needs to be so complicated... is there a reason it needs to be done like this as opposed to just having a single regex?